common/djangoapps/nodebb/signals/handlers.py

```diff
@@ -0,0 +1,42 @@
+from logging import getLogger
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from common.lib.nodebb_client.client import NodeBBClient
+from xmodule.modulestore.django import modulestore
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseEnrollment
+
+log = getLogger(__name__)
+
+
+@receiver(post_save, sender=CourseOverview, dispatch_uid="nodebb.signals.handlers.create_category_on_nodebb")
+def create_category_on_nodebb(sender, instance, created, **kwargs):
```

Comment:
I think this handler should call a celery task and that task should make relevant API calls to nodebb.